apollo/models/agent.py

```diff
@@ -16,6 +17,19 @@ class Agent(Base):
     oauth_client = relationship('OAuthClient', uselist=False,
                                 cascade="all, delete-orphan")
 
+    @property
+    def connection_status(self):
+        try:
+            return str(
+                WebSocketManager().connections[self.id].client_state).split(
```

Comment:
```
return str(
    WebSocketManager().connections[self.id].client_state
).split('.')[1].lower() is None
```

Although the `is None` confuses me here.

In the end though I'd just return the enum value rather than doing a string conversion to then covert it to your own enum later again.